package/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "@transmitsecurity/riskid-reactjs-ts",
-    "version": "1.0.14",
+    "version": "1.0.15",
```

Comment:
please update `yarn.lock` as well

Reply:
i don't think we need to update this in this case, it doesn't contain reference to the package name/version